### PR TITLE
Manage request: list and delete request

### PR DESCRIPTION
**Pull request by ducminhvu**

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -271,3 +271,13 @@ input {
 .wrapper {
     text-align: center;
```
Comment (khanhkhanh) [inline app/assets/stylesheets/custom.css.scss]: Indentation

File: app/views/admin/requests/_request.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<li id="request-<%= request.id %>">
+  <h3><%= t("admin.request.book_title") + request.book_name %></h3>
```
Comment (khanhkhanh) [inline app/views/admin/requests/_request.html.erb]: Try removing ()s

Comment (ducminhvu) [inline app/views/admin/requests/_request.html.erb]: can not remove ()

File: app/models/request.rb [hunk 3]
```diff
@@ -1,3 +1,6 @@
 class Request < ActiveRecord::Base
   belongs_to :user
+
+  scope :requesting, ->{where bought: false}
+  default_scope ->{order(bought: :asc, created_at: :desc)}
```
Comment (dieunb) [inline app/models/request.rb]: don't use `default_scope`

File: app/views/admin/dashboards/index.html.erb [hunk 4]
```diff
@@ -2,3 +2,11 @@
 <h1><%= t "admin.admin_title" %></h1>
 <h3><%= link_to t("admin.manage_users"), admin_users_path %></h3>
 <h3><%= link_to t("admin.manage_books"), "#" %></h3>
+<h3>
+  <%= link_to admin_requests_path do %>
+    <%= t "admin.manage_request" %>
+    <span class="badge">
+      <%= "#{Request.requesting.count}" unless Request.requesting.count == 0 %>
```
Comment (dieunb) [inline app/views/admin/dashboards/index.html.erb]: `0 == Request.requesting.count`

File: app/models/request.rb [hunk 5]
```diff
@@ -1,3 +1,6 @@
 class Request < ActiveRecord::Base
   belongs_to :user
+
+  scope :list_request, ->{order(bought: :asc, created_at: :desc)}
```
Comment (dieunb) [inline app/models/request.rb]: remove `()`

File: app/views/admin/dashboards/index.html.erb [hunk 6]
```diff
@@ -2,3 +2,13 @@
 <h1><%= t "admin.admin_title" %></h1>
 <h3><%= link_to t("admin.manage_users"), admin_users_path %></h3>
 <h3><%= link_to t("admin.manage_books"), "#" %></h3>
+<h3>
+  <%= link_to admin_requests_path do %>
+    <%= t "admin.manage_request" %>
+    <% if Request.requesting.count >0 %>
+      <span class="badge">
+        <%= "#{Request.requesting.count}" %>
```
Comment (dieunb) [inline app/views/admin/dashboards/index.html.erb]: why not `<%= Request.requesting.count %>`

File: app/views/admin/requests/_request.html.erb (hunk 2 above)
Comment (dieunb) [inline app/views/admin/requests/_request.html.erb]: try use i18n variable.

**Pull request merged**